src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -75,9 +75,102 @@ public SimpleConfigModel()
 
         public IReadOnlyList<ICreationPathModel> PrimaryOutputs { get; set; }
 
-        public IReadOnlyDictionary<string, string> Tags { get; set; }
+        private IReadOnlyDictionary<string, string> _tagsDeprecated { get; set; }
```

Comment:
If this needs to be a property, it should be cased as a method (`TagsDeprecated`), if it can be made to be a field (since it's private now) the name's OK